src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java

```diff
@@ -355,6 +367,22 @@ private void createRequestToDeleteInstance(URIBuilder uriBuilder) throws DicomFu
     }
   }
 
+  private void createRequestToCreateDicomStore(URIBuilder uriBuilder) throws DicomFuseException {
```

Comment:
The naming here is a bit confusing... you're actually executing the request here, not just creating it. How about just moving this entire method into the createDicomStore method above?

Reply:
Done